### ISSUE TITLE
ocabot migration duplicating lines in migration issue?

### ISSUE THREAD

**Issue by sbidoul** [OCA/oca-github-bot#214]
See for example https://github.com/OCA/server-env/issues/120 where the issue was created correctly but now lines with references to a PR are duplicated.

**Comment by pedrobaeza** [comment]
I have observed it in some cases, but I'm not able to deduce what is reason. A problem in the regex?

**Comment by sbidoul** [comment]
Another example: https://github.com/OCA/partner-contact/issues/1353

**Comment by eLBati** [comment]
Something similar is still happening:

https://github.com/OCA/l10n-italy/pull/3026#issuecomment-1311379301
produced

![image](https://user-images.githubusercontent.com/1033131/201300271-2c1c70eb-cfe1-4cdb-978b-fb9f5b27a06a.png)

**Comment by SirTakobi** [comment]
> Something similar is still happening:
> 
> [OCA/l10n-italy#3026 (comment)](https://github.com/OCA/l10n-italy/pull/3026#issuecomment-1311379301) produced
> 
> ![image](https://user-images.githubusercontent.com/1033131/201300271-2c1c70eb-cfe1-4cdb-978b-fb9f5b27a06a.png)

That is because the alphabetical order of modules is checked in https://github.com/OCA/oca-github-bot/blob/44ad4a5be6c19f3722566bd1082b880c003e6910/src/oca_github_bot/tasks/migration_issue_bot.py#L60: when the bot reads `l10n_it_ipa`, it adds (before `l10n_it_ipa`) the `l10n_it_declaration_of_intent` line because `'l10n_it_ipa' > 'l10n_it_declaration_of_intent'`.

**TL;DR**: the bot misbehaves when the modules are not written in alphabetical order, should we fix it?

**Comment by pedrobaeza** [comment]
It's not the same. This bot command assumes an alphabetical order in the migration issue, which is not the case.

**Comment by pedrobaeza** [comment]
You should order the modules IMO instead of over-complicating the bot.

**Comment by eLBati** [comment]
Thanks, ordered
